README.md

```diff
@@ -173,54 +173,60 @@ A function that transforms the parsed configuration. Receives the result object
 
 ##### configPath
 
 Type: `string`
 
-If provided, cosmiconfig will load and parse a config from this path, and will not perform its usual search.
+If provided, cosmiconfig's [`load()`](#loadconfigpath) will default to loading a config from this path.  Passing a string parameter will override this option.
 
 ##### format
 
 Type: `'json' | 'yaml' | 'js'`
 
-The expected file format for the config loaded from `configPath`.
+The expected file format for the config when running [`load()`](#loadconfigpath).
 
 If not specified, cosmiconfig will try to infer the format using the extension name (if it has one).
 In the event that the file does not have an extension or the extension is unrecognized, cosmiconfig will try to parse it as a JSON, YAML, or JS file.
 
 ### Instance methods (on `explorer`)
 
-#### `load([searchPath, configPath])`
+#### `search([searchPath])`
 
-Find and load a configuration file. Returns a Promise that resolves with `null`, if nothing is found, or an object with two properties:
+Searches for a configuration file. Returns a Promise that resolves with `null`, if nothing is found, or an object with two properties:
 - `config`: The loaded and parsed configuration.
 - `filepath`: The filepath where this configuration was found.
 
-You should provide *either* `searchPath` *or* `configPath`. Use `configPath` if you know the path of the configuration file you want to load. Note that `configPath` takes priority over `searchPath` if both parameters are specified.
-
 ```js
-explorer.load()
-
-explorer.load('start/search/here');
-explorer.load('start/search/at/this/file.css');
-
-explorer.load(null, 'load/this/file.json');
+explorer.search() // searches from process.cwd()
+explorer.search('start/search/here');
+explorer.search('start/search/at/this/file.css');
 ```
 
 If you provide `searchPath`, cosmiconfig will start its search at `searchPath` and continue to search up the directory tree, as documented above.
 By default, `searchPath` is set to `process.cwd()`.
 
-If you provide `configPath` (i.e. you already know where the configuration is that you want to load), cosmiconfig will try to read and parse that file. Note that the [`format` option](#format) is applicable for this as well.
+#### `load([configPath])`
+
+Loads a configuration file directly.  Rejects with an error if the file does not exist, if the file contents are empty, or if the file cannot be parsed.  Upon success it returns a Promise that resolves with an object with two properties:
+- `config`: The loaded and parsed configuration.
+- `filepath`: The filepath where this configuration was found.
+
+```js
+explorer.load() // loads options.configPath
+explorer.load('load/this/file.json');
+```
+
+Note the [`format` option](#format) is applicable for this.
 
 #### `clearFileCache()`
```

Comment:
Shall we rename these to `clearLoadCache` and `clearSearchCache`?

Reply:
👍

Reply:
I guess they're synonymous i.e. the loadCache will only ever cache files and likewise with search <--> directories so renaming makes sense.  I'll push that change soon.